frontend/src/utils/analysis-utils.ts

```diff
@@ -111,6 +111,7 @@ const operations = {
     const ceil = sortedValues.length / 2;
     return (sortedValues[floor] + sortedValues[ceil]) / 2;
   },
+  intersect_percentage: (data: number[]) => mean(data),
```

Comment:
I am not sure about calculation required for intersect_precentage. Let me know if we need to change it.

Reply:
Hmm I think we need to error our in this case or at least raise a warning. With just the percentage we can't give an accurate number here from the frontend. But this shouldn't be triggered atm as the analysis is run by the backend, no?

Reply:
I think I don't really understand what is the usage of this operator. I can see it is used by the loadFeaturesClientSide function that is called by the fetchImpactLayerData. I was thinking it was used to display the layer with the selected statistic but it seems not. If I change with "intersect_percentage: () => null" it continues working fine. I change this line because of typescript but I don't know the usage/impact and what to do with it.

Reply:
So if I recall correctly this is used to make the analysis calculations locally, if you don't have access to a stats API.
It is less relevant now but at the beginning, some deployments wanted to be "independent" from the API.

Reply:
ok understood ! I change to raise an error if trying to calculate intersect_percentage from client side then